src/main/java/org/sonatype/repository/conan/internal/proxy/ConanProxyFacet.java

```diff
@@ -130,33 +140,50 @@ private Content putPackage(final String assetPath,
   private Content putMetadata(final String assetPath,
                               final Content content,
                               final AssetKind assetKind,
+                              final String group,
                               final String project,
-                              final String version,
-                              final String group)
+                              final String version)
       throws IOException
   {
+    TempBlob updatedBlob = null;
     StorageFacet storageFacet = facet(StorageFacet.class);
     try (TempBlob tempBlob = storageFacet.createTempBlob(content.openInputStream(), HASH_ALGORITHMS)) {
       AttributesMap attributesMap;
-      TempBlob blob = tempBlob;
       switch (assetKind) {
         case DOWNLOAD_URL:
-          blob = absoluteUrlRemover.removeAbsoluteUrls(tempBlob, getRepository());
-          // break intentionally left out!
+          String indexAssetName = getProjectIndexName(group, project, version);
+          updatedBlob = absoluteUrlRemover.updateAbsoluteUrls(tempBlob, getRepository(), indexAssetName);
```

Comment:
If you store rewritten absolute paths like this and the Nexus base url changes then all these files will be incorrect and the repo will be broken. For absolute paths it's safer to do the rewrites on an ad-hoc basis unless there are performance reasons not to, particularly if there's no way to force a manual rebuild of all these in the unlikely event someone changes the base url.

Reply:
I have raised this as a new ticket https://github.com/sonatype-nexus-community/nexus-repository-conan/issues/12 rather than complicate this PR